batch-submitter/Dockerfile

```diff
@@ -2,12 +2,12 @@ FROM golang:1.18.0-alpine3.15 as builder
 
 RUN apk add --no-cache make gcc musl-dev linux-headers git jq bash
 
-COPY ./l2geth /l2geth
-COPY ./go/bss-core /go/bss-core
-COPY ./go/batch-submitter/go.mod ./go/batch-submitter/go.sum /go/batch-submitter/
+COPY ./batch-submitter /go/batch-submitter
+COPY ./bss-core /go/bss-core
+COPY ./l2geth /go/l2geth
+COPY ./batch-submitter/docker.go.work /go/go.work
+
 WORKDIR /go/batch-submitter
-RUN go mod graph | grep -v l2geth | grep -v bss-core | awk '{if ($1 !~ "@") print $2}' | xargs -n 1 go get
-COPY ./go/batch-submitter/ ./
```

Comment:
Not sure if you removed this intentionally, but if we can get rid of magic like that, awesome.